modules/manually-rotating-cloud-creds.adoc

```diff
@@ -66,10 +90,24 @@ You can also use the command line interface to complete all parts of this proced
 +
 [source,terminal]
 ----
-$ oc -n openshift-cloud-credential-operator get CredentialsRequest -o json | jq -r '.items[] | select (.spec.providerSpec.kind=="<provider_spec>") | .spec.secretRef'
+$ oc -n openshift-cloud-credential-operator get CredentialsRequest \
+-o json | jq -r '.items[] | select (.spec.providerSpec.kind=="<provider_spec>") | .spec.secretRef' <1>
 ----
 +
-Where `<provider_spec>` is the corresponding value for your cloud provider: `AWSProviderSpec` for AWS, `AzureProviderSpec` for Azure, or `GCPProviderSpec` for GCP.
+<1> For `<provider_spec>`, specify the corresponding value for your cloud provider:
++
+--
+* `AWSProviderSpec` for AWS
```

Comment:
I guess it's a matter of taste, but I've always done

```
`value`: word
...
```

Reply:
oh - yeah I just added line breaks to the existing content but yes that is a much better format :D